pkg/printers/internalversion/printers.go

```diff
@@ -41,6 +41,7 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
```

Comment:
It is one constant. Copy it and put a comment where it is from. Otherwise this postpones the work of fixing the dependencies to later and delgates it to somebody else. Neither is good.